### PR TITLE
Translation maintenance improvements

### PR DESCRIPTION
**Pull request by paulwarren-wk**
This PR changes how translations are managed in order to enable automated scanning for new translation keys that need to be translated.

Translations are now split out into multiple files corresponding to different `i18next` namespaces:

* `translation` - which contains all static translation keys
* `currencies` - translations of currency names and symbols
* `referenceparts` - translations of reference part names

The new npm `i18n` script (`npm run i18n`) will scan JS and HTML files for static translation keys, add any missing keys to the `translation.json` file and move any unused keys to `translation_old.json`.

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by sagesmith-wf — APPROVED** [review]

**Review by derekgengenbacher-wf — DISMISSED** [review]

**Review by brettkail-wk — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
@paulwarren-wk You have merge conflicts on this PR.

File: iXBRLViewerPlugin/viewer/src/i18n/es/translation.json [hunk 1]
```diff
@@ -22,9 +22,13 @@
     "highlight": "Destacar",
     "factProperties": "Propiedades del hecho",
     "footnoteProperties": "Propiedades de la nota al pie de página",
-    "reset": "",
-    "filter": "",
-    "conceptType": "",
+    "documentOutline": "",
+    "searchTitle": "Búsqueda de hechos",
```
Comment (sagesmith-wf) [inline iXBRLViewerPlugin/viewer/src/i18n/es/translation.json:26]: Sorting the keys seems worthwhile to me

**Review by sagesmith-wf — DISMISSED** [review]

File: iXBRLViewerPlugin/viewer/src/i18n/es/translation.json (hunk 1 above)
Comment (paulwarren-wk) [reply iXBRLViewerPlugin/viewer/src/i18n/es/translation.json]: Done in d93b3c4

**Review by brettkail-wk — APPROVED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 Unit and Skynet tests pass successfully. Builds successfully. Built successful locally and verified no regressions.

@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**